djangocms_text_ckeditor/html.py

```diff
@@ -6,44 +6,41 @@
 import html5lib
 from django.utils.module_loading import import_string
 from django.utils.six import BytesIO
-from html5lib import sanitizer, serializer, treebuilders, treewalkers
+from html5lib import serializer, treebuilders, treewalkers
+from html5lib.filters import sanitizer
+from html5lib.constants import namespaces
 from PIL import Image
 
 from . import settings
 from .sanitizer import TextSanitizer
 from .utils import plugin_to_tag
 
 
-def _get_default_parser():
-    opts = {}
-
-    sanitizer.HTMLSanitizer.acceptable_elements.extend(['cms-plugin'])
+def _filter_kwargs():
+    kwargs = dict(
+        allowed_elements=set(
+            tuple(sanitizer.allowed_elements) + ((namespaces['html'], 'cms-plugin'),)))
 
     if settings.TEXT_HTML_SANITIZE:
-        sanitizer.HTMLSanitizer.acceptable_elements.extend(
+        kwargs['allowed_elements'] = set(
```

Comment:
When I add `'TEXT_ADDITIONAL_TAGS': ('iframe',),` to test_settings. It won't work. 

Steps to reproduce:

- create page with text plugin
- click `Source` in ckeditor, paste `<iframe src="https://www.divio.com/en/"></iframe>`
- unclick `Source` to go back to normal editing in ckeditor
- click `Save`
- on page should show iframe, not text `<iframe src="https://www.divio.com/en/"></iframe>`

Please check also for ADDITONAL_ATTRIBUTES and ADDITIONAL_PROTOCOLS if it's works properly.

Reply:
Ideally you can add unit test for that

Reply:
This issue exists now after I updated, rolling back